src/endpoint/Timesheet.Delete/FlowStep/DateTimesheetFlowStep.cs

```diff
@@ -0,0 +1,44 @@
+using System.Globalization;
+
+namespace GarageGroup.Internal.Timesheet;
+
+internal static partial class DateTimesheetFlowStep
```

Comment:
Rename to `TimesheetDeleteFlowStep`. Find unused fields and consts and remove them.